server/core/config/raw/metrics.go

```diff
@@ -28,7 +28,7 @@ func (s *Statsd) Validate() error {
 	return validation.ValidateStruct(s,
 		validation.Field(&s.Host, validation.Required),
 		validation.Field(&s.Port, validation.Required),
-		validation.Field(&s.Host, is.IP),
```

Comment:
Why are you removing the ip? and not just adding the Host

Reply:
`IsHost` includes `IsIP` condition, so I removed `IsIP` validation.

https://github.com/asaskevich/govalidator/blob/f21760c49a8d602d863493de796926d2a5c1138d/validator.go#L851-L853

Reply:
@jamengual  Would you review this PR?